API/AddContact.php

```diff
@@ -0,0 +1,48 @@
+<?php
+	$inData = getRequestInfo();
+	
+    $firstName = $inData["firstName"];
+    $lastName = $inData["lastName"];
+    $email = $inData["email"];
+    $phone = $inData["phone"];
+
+    //connection to database
+	$servername = "localhost";
+    $username = "username";
+    $password = "password";
+    $dbname = "myDB";
+
+    // Create connection
+    $conn = new mysqli($servername, $username, $password, $dbname);
+	if ($conn->connect_error) 
+	{
+		returnWithError( $conn->connect_error );
+	} 
+	else
+	{
+		$stmt = $conn->prepare("INSERT into Contacts (ID, firstName, lastName, email, phone, userID) VALUES(NULL, ?, ?, ?, ?, NULL)");
+		$stmt->bind_param("ssss", $firstName, $lastName, $email, $phone);
```

Comment:
Need to add userID for SQL query.
 ```suggestion
		$stmt->bind_param("ssssi", $firstName, $lastName, $email, $phone, $userID);
```

Reply:
We also need to store the date created for contacts (per the requirements in the intro to the course slides).